deploy/example/pv-smb-krb5.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  annotations:
+    pv.kubernetes.io/provisioned-by: smb.csi.k8s.io
+  name: pv-smb-krb5
+spec:
+  capacity:
+    storage: 100Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: smb
+  mountOptions:
+    - sec=krb5
+    - cruid=1000
+    - seal
+    - vers=3.0
+    - nosuid
+    - noexec
+    - dir_mode=0777
+    - file_mode=0777
+    - uid=1001
+    - gid=1001
+    - noperm
+    - mfsymlinks
+    - cache=strict
+    - noserverino  # required to prevent data corruption
+  csi:
+    driver: smb.csi.k8s.io
+    readOnly: false
+    # volumeHandle format: {smb-server-address}#{sub-dir-name}#{share-name}
+    # make sure this value is unique for every share in the cluster
+    volumeHandle: smb-server.default.svc.cluster.local/share##
+    volumeAttributes:
+      source: "//smb-server.default.svc.cluster.local/share"
+    nodeStageSecretRef:
```

Comment:
I think using a storage class example is better

Reply:
Done